ambry-api/src/main/java/com/github/ambry/account/Container.java

```diff
@@ -136,6 +137,21 @@
    */
   public static final ContainerStatus UNKNOWN_CONTAINER_STATUS = ContainerStatus.ACTIVE;
 
+  /**
+   * The last update time of {@link #UNKNOWN_CONTAINER}
+   */
+  public static final Long UNKNOWN_CONTAINER_LAST_UPDATE_TIME = System.currentTimeMillis();
+
+  /**
+   * The last update time of {@link #DEFAULT_PUBLIC_CONTAINER}
+   */
+  public static final Long DEFAULT_PUBLIC_CONTAINER_LAST_UPDATE_TIME = System.currentTimeMillis();
```

Comment:
This will set the default time to whatever the timestamp was when this classes static initialization completed. Maybe the default time should be 0 for consistency? Also, generally prefer long over Long when there is no need for a primitive to be null and no need to use the long as part of a generic. This avoids unnecessary unboxing penalties.

Reply:
Updated the type to long and change the default time to 0 for consistency.